core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
I would like to get input from other contributors about the style used here. Is the `:` on the next line and two-space alignment enforced by `clang-format`, or can we have something like this instead?
```
BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) :
	nodes(p_nodes),
 	planes(p_planes),
	aabb(p_aabb),
	error_radius(p_error_radius) {
}
```

My main concern with the current style is that mixing tabs and spaces for indentation + alignment is (IMO) a very bad practice, though sadly it's one `clang-format` tends to encourage as it was designed from ground up for space-based indentation + alignment.